projects/planner/src/app/plan/diveoptions/diveoptions.component.ts

```diff
@@ -240,6 +240,17 @@ export class DiveOptionsComponent extends Streamed implements OnInit {
 
     public switchStopsRounding(): void {
         this.options.roundStopsToMinutes = !this.options.roundStopsToMinutes;
+        if (this.options.roundStopsToMinutes){
```

Comment:
What is the reason to allow only one of these switches?
Shouldn't be better to use enum?
As per my recommendation in the ticket, stick to only one switch and round to the runtime only.